src/cocoa/toga_cocoa/dialogs.py

```diff
@@ -66,7 +64,7 @@ def error(window, title, message):
 def stack_trace(window, title, message, content, retry=False):
     alert = NSAlert.alloc().init()
     alert.icon = window.app.icon._impl.native
-    alert.setAlertStyle_(NSCriticalAlertStyle)
+    alert.setAlertStyle_(NSAlertStyle.Critical)
```

Comment:
Is there a missing `.value` here?

Reply:
There is. Just corrected.

Reply:
Are you sure the `.value` is required? It doesn't *hurt* to have it - but Rubicon should convert Python Enums into their underlying value types... if it doesn't, then that's a bug in Rubicon.

Reply:
I didn't know that Rubicon did that :)

Reply:
Me neither :-p